### ISSUE TITLE
Change oidc redirect URI validation

### ISSUE THREAD

**Issue by phavekes** [OpenConext/OpenConext-manage#309]
[This issue is imported from pivotal - Originaly created at Jul 15, 2019 by Bart Geesink](https://www.pivotaltracker.com/story/show/167285472)

The validation currently rejects URI\'s like this:
nl.uva.myuva://redirect

I\'d like to be able to support such redirect URIs, for support of mobile apps

**Comment by phavekes** [comment]
I get stacktraces when I try to add the custom redirect URI nl.uva.myuva://redirect

java.lang.IllegalArgumentException: No validation defined for uri 

Afterwards, it does save, and push works as well. 
I only have this the test environment (running 4.0.6) not on test2. (Bart Geesink - Jul 18, 2019)

**Comment by phavekes** [comment]
The metadata_configuration JSON schema templates in OpenConext-deploy are environment independent. The `manage.oidc_rp_redirect_url_format` group_vars variable determines the `format` for the `redirectUrls` in OpenConext-deploy/roles/manage-server/files/metadata_configuration/oidc10_rp.schema.json.j2. The `uri` validation in Manage was added / deployed in 4.0.7-SNAPSHOT and is not present in 4.0.6. I have released version 4.0.7 of manage this morning. 

The solution is to deploy 4.0.7 to the test environment or revert the variable `manage.oidc_rp_redirect_url_format` to `url` in OpenConext-deploy/environments-external/test/group_vars/test.yml. (Okke Harsta - Jul 19, 2019)
